update-pulumi-terraform/update-pulumi-terraform.sh

```diff
@@ -6,7 +6,7 @@ set -o pipefail
 # Note the ordering here is designed to prevent problems hitting the "big 3"
 # providers by not doing them first.
 PROVIDERS="digitalocean packet newrelic cloudflare linode f5bigip newrelic "
-PROVIDERS+="random vsphere openstack gcp azure azuread aws gitlab"
+PROVIDERS+="random vsphere openstack gcp azure azuread aws gitlab mysql postgresql"
```

Comment:
I think the intention was to do gcp, azure, azuread and aws last, FWIW.

Reply:
Good point! Changing it now